RELEASE-NOTES.md

```diff
@@ -1,5 +1,22 @@
 # Prefect Release Notes
 
+## Release 2.19.3
+
+### Enhancements
+- Provide URL in CLI output upon work pool creation — https://github.com/PrefectHQ/prefect/pull/13597
+- Allow flow parameter schema generation when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/13620
+
+### Fixes
+- Skip on cancellation hooks if runner can't load flow — https://github.com/PrefectHQ/prefect/pull/13669
```

Comment:
```suggestion
- Skip on cancellation hooks if runner can't load flow — https://github.com/PrefectHQ/prefect/pull/13669
```
Maybe "Ensure graceful cancellation of flow runs corresponding to deleted deployments" ?